### PR TITLE
Add missing analytical events and add design-rush icon in footer section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**

### CODE REVIEW

File: vue-frontend/src/pages/jobs/[id].vue [hunk 1]
```diff
@@ -382,7 +382,7 @@ export default {
   @apply tw-flex tw-items-baseline;
 }
 :deep(ul > li > a > svg) {
-  @apply tw-flex tw-items-center;
+  @apply tw-flex tw-items-center tw-w-[23px] tw-h-[25px] tw-ml-[1.5px];
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/jobs/[id].vue:385]: In width < 438, let's make 2 rows each containing 4 icons

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/pages/portfolio/index.vue [hunk 2]
```diff
@@ -6,7 +6,7 @@
       </template>
     </metainfo>
     <ScreenHeaderV2 />
-    <Portfolio />
+    <Portfolio ref="portfoliopage" />
```
Comment (cp-shruti-s) [inline vue-frontend/src/pages/portfolio/index.vue:9]: I think we don't need add over here !!

Comment (cp-priyanka-g) [reply vue-frontend/src/pages/portfolio/index.vue:9]: Sumita told "Its correct".

File: vue-frontend/src/pages/jobs/[id].vue (hunk 1 above)
Comment (cp-sumi-k) [inline vue-frontend/src/pages/jobs/[id].vue:385]: Why do we need this? Height and width are not used. Refer attachment below

Screenshot from 2022-10-18 16-44-41

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**